### PR TITLE
Add some of the im-pipelines tools to master

### PR DESCRIPTION
**Pull request by simonbray**
Some of the tools from the im-pipelines repo (see here: https://github.com/InformaticsMatters/pipelines/tree/master/src/python/pipelines/rdkit). Thanks a lot to @lorrainealisha75 for working on this.

### CODE REVIEW

File: chemicaltoolbox/im-pipelines/o3dAlign.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            -e '$emin'
+            --meta
+            -of sdf
+            -o outp &>> $logfile &&
+        cat outp_metrics.txt &>> $logfile &&
+        gzip -d outp.sdf.gz
+    ]]></command>
+    <inputs>
+        <param name="infile" type="data" format="sdf" label="Input file" help="Input file in SDF format"/>
+        <param name="molquery" type="data" format="mol" label="Mol query file" help="Mol file to align against"/>
```
Comment (bgruening) [inline chemicaltoolbox/im-pipelines/o3dAlign.xml:29]: mol or sdf?

File: chemicaltoolbox/im-pipelines/o3dAlign.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <param name="qmolidx" type="integer" value="1" label="Query molecule index" help="Query molecule index in SD-file if not the first"/>
+        <param name="threshold" type="float" value="1" label="Threshold" help="Score cutoff relative to alignment of query to itself"/>
```
Comment (bgruening) [inline chemicaltoolbox/im-pipelines/o3dAlign.xml:31]: add min/max

File: chemicaltoolbox/im-pipelines/o3dAlign.xml [hunk 3]
```diff
@@ -0,0 +1,103 @@
+<tool id="ctb_im_o3dAlign" name="Open 3D Align" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
```
Comment (bgruening) [inline chemicaltoolbox/im-pipelines/o3dAlign.xml:1]: only lower case letter in IDs?

**Comment by bgruening** [comment]
:+1: from my side, when you fix the one tool-ID.

Please feel free to merge with the new release.

**Comment by simonbray** [comment]
That's great, thanks a lot!

File: chemicaltoolbox/im-pipelines/pbf_ev.xml [hunk 4]
```diff
@@ -24,7 +24,6 @@
     <tests>
         <test>
             <param name="infile" value="Kinase_inhibs.sdf" ftype="sdf"/>
-            <output name="outfile" ftype="sdf" file="pbf_ev_output1.sdf" lines_diff="72"/>
```
Comment (bgruening) [inline chemicaltoolbox/im-pipelines/pbf_ev.xml]: what was the problem here?

Comment (bgruening) [reply chemicaltoolbox/im-pipelines/pbf_ev.xml]: Ok, if we do not have a real problem here, let's merge that.

**Comment by bgruening** [comment]
@simonbray I added the xcos tool real quick. That should get Tim started.

**Comment by bgruening** [comment]
@simonbray should this go in?

@tdudgeon I added xcos here already.

**Comment by simonbray** [comment]
Yes, I'm in favour.

**Pull request merged**